apps/drupal/templates/block/block--system-branding-block.html.twig

```diff
@@ -8,25 +8,19 @@
  * enabled in the block configuration.
  *
  * Available variables:
+ * - url: The url for the front page of the site.
  * - site_logo: Logo for site as defined in Appearance or theme settings.
+ * - logo_inline_svg: The svg logo if it exists.
+ * - logo_img_src: The src logo if it exists.
  * - site_name: Name for site as defined in Site information settings.
  * - site_slogan: Slogan for site as defined in Site information settings.
  */
 #}
 
-
 {% block content %}
 
-  <h1>i r branding nerp</h1>
-
-  {% include '@atoms/button/_button.twig' with {
-    button_size: 'lg',
-    button_text: 'hello',
+  {% include "@atoms/_branding.twig" with {
+    logo_inline_svg: '@atoms/image/logo.svg',
```

Comment:
Hey, since this is the Drupal block, bring over the if uploaded checks just like in the demo of this in PL.